buildkitd/dockerd-wrapper.sh

```diff
@@ -24,7 +24,17 @@ execute() {
         exit 1
     fi
 
-    start_dockerd
+    # Sometimes, when dockerd starts containerd, it doesn't come up in time. This timeout is not configurable from
+    # dockerd, therefore we retry... since most instances of this timeout seem to be related to networking or scheduling
+    # when many WITH DOCKER commands are also running. Logs are printed for each failure.
+    for i in 1 2 3 4 5; do
+      if start_dockerd; then
```

Comment:
nit: indentation is inconsistent with the rest of the file (which uses 4 spaces).